mlir/test/Dialect/Ptr/canonicalize.mlir

```diff
@@ -13,3 +13,61 @@ func.func @zero_offset(%ptr: !ptr.ptr<#ptr.generic_space>) -> !ptr.ptr<#ptr.gene
   %res0 = ptr.ptr_add %ptr, %off : !ptr.ptr<#ptr.generic_space>, index
   return %res0 : !ptr.ptr<#ptr.generic_space>
 }
+
```

Comment:
nit: I would add `-split-input-file`.

Reply:
AFAIK that's only needed if there are multiple modules separated by a `// -----`, and currently there's only one module in the test.